docs/Supported-Platforms.md

```diff
@@ -92,10 +92,11 @@ builds ([brim/1266](https://github.com/brimdata/brim/issues/1266)).
 ## Linux
 
 Brim's [test automation](#automated-testing) runs on Ubuntu 18.04 (`.deb`
-package) and CentOS 8 (`.rpm` package) and therefore these are the platforms
-on which we are best able to ensure quality and prevent regressions.
+package) and therefore this is the Linux distribution on which we are best able
+to ensure quality and prevent regressions. 
```

Comment:
I corrected this statement to accurately reflect that the CI no longer tests on CentOS. See [https://github.com/brimdata/brim/pull/1985#discussion_r766045844](https://github.com/brimdata/brim/pull/1985#discussion_r766045844) .